plugins/catalog-backend/migrations/20200527114117_location_update_log_latest_view.js

```diff
@@ -33,6 +33,7 @@ exports.up = async function up(knex) {
     ) t2
     ON t1.location_id = t2.location_id
     AND t1.created_at = t2.MAXDATE
+    GROUP BY t1.location_id
```

Comment:
Just noting that those that have existing persistent databases won't be picking this up. Are we at the point where we can't retrofit migrations yet? Not sure

Reply:
This is basically override for the override that we merged couple of weeks ago, should be okay. But - fair point, maybe we need to settle on migrations workflow going forward

Reply:
On a second thought, we already have at least couple of people that I'm aware of that do use persistent dbs. Will think it over and bring it tomorrow during sync ;)

Reply:
Didnt fully settle but last week we kinda arrived at that we want to only append mugrations for now, but then flatten once we do a beta release or something like it